configs/uc-davic.nrel-op.json

```diff
@@ -0,0 +1,83 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://ca-ebike-openpath.nrel.gov",
```

Comment:
The `connectUrl` should end with `/api`, as in 
https://github.com/e-mission/nrel-openpath-deploy-configs/blob/1eaadf8a6433a6f282dff09ed055c08060b807e8/configs/cortezebikes.nrel-op.json#L6